.github/workflows/devtools-workflow.yaml

```diff
@@ -1,7 +1,13 @@
 name: Dev tools workflow
 on:
   pull_request:
-      
+    types:
+      - unlabeled
+      - labeled
+      - synchronize
+      - opened
+      - reopened
+
```

Comment:
I recommend using the same path-ignore as the main workflow. Because we don't always want the version to be updated.

Reply:
You're right, it should be fixed now :)